source/material_model/rheology/elasticity.cc

```diff
@@ -228,6 +239,16 @@ namespace aspect
             out.additional_outputs.push_back(
               std::make_unique<ElasticAdditionalOutputs<dim>> (n_points));
           }
+
+        // Create the ReactionRateOutputs that are necessary for the operator splitting
```

Comment:
Should we enforce that operator splitting is used? Or maybe this enforced somewhere below?

Reply:
Yep, I've added that.